src/helm/benchmark/run_specs.py

```diff
@@ -625,6 +652,114 @@ def get_instruction_following_critique_metric_specs(num_respondents: int) -> Lis
+        ),
+    ]
+
+
+def get_cleva_generative_task_metric_spec(task: str, subtask: Optional[str], **kwargs) -> List[MetricSpec]:
+    CLEVA_GEN_TASK_TO_METRIC: Dict[str, Callable] = {
+        "opinion_mining:opinion_target_extraction": get_exact_match_metric_specs,
+        "paraphrase_generation": get_cleva_paraphrase_generation_metric_specs,
+        "closed_book_question_answering:generative_question_answering": get_exact_match_metric_specs,
+        "conceptual_generalization": get_cleva_topk_accuracy_metric_specs,
+        "translation:en2zh": get_cleva_machine_translation_metric_specs,
+        "translation:zh2en": get_cleva_machine_translation_metric_specs,
+        "mathematical_calculation:add": get_exact_match_metric_specs,
+        "mathematical_calculation:sub": get_exact_match_metric_specs,
+        "mathematical_calculation:mul": get_exact_match_metric_specs,
+        "inductive_reasoning:add": get_exact_match_metric_specs,
+        "inductive_reasoning:sub": get_exact_match_metric_specs,
+        "inductive_reasoning:mul": get_exact_match_metric_specs,
+        "reasoning_primitive:dyck_language": get_exact_match_metric_specs,
+        "reasoning_primitive:pattern_induction": get_exact_match_metric_specs,
+        "reasoning_primitive:pattern_matching": get_exact_match_metric_specs,
+        "reasoning_primitive:variable_sub": get_exact_match_metric_specs,
+        "subject_knowledge:art": get_exact_match_metric_specs,
+        "subject_knowledge:biomedicine": get_exact_match_metric_specs,
+        "subject_knowledge:chemistry": get_exact_match_metric_specs,
+        "subject_knowledge:computer_science": get_exact_match_metric_specs,
+        "subject_knowledge:economics": get_exact_match_metric_specs,
+        "subject_knowledge:geography": get_exact_match_metric_specs,
+        "subject_knowledge:history": get_exact_match_metric_specs,
+        "subject_knowledge:law": get_exact_match_metric_specs,
+        "subject_knowledge:literature": get_exact_match_metric_specs,
+        "subject_knowledge:math": get_exact_match_metric_specs,
+        "subject_knowledge:other_general": get_exact_match_metric_specs,
+        "subject_knowledge:philosophy": get_exact_match_metric_specs,
+        "subject_knowledge:physics": get_exact_match_metric_specs,
+        "subject_knowledge:politics": get_exact_match_metric_specs,
+        "summarization:dialogue_summarization": partial(get_basic_metric_specs, ["chinese_rouge_2"]),
+        "pinyin_transliteration:pinyin2zh": partial(get_basic_metric_specs, ["chinese_bleu_1"]),
+        "pinyin_transliteration:zh2pinyin": partial(get_basic_metric_specs, ["chinese_bleu_1"]),
+        "dialogue_generation:task_oriented": partial(get_basic_metric_specs, ["chinese_bleu_1"]),
+        "data_to_text_generation": partial(get_basic_metric_specs, ["chinese_bleu_1"]),
+        "mathematical_reasoning:math_world_problem": partial(get_basic_metric_specs, ["cleva_math_result_match"]),
+    }
```

Comment:
Could you create a new file `src/helm/benchmark/presentation/cleva_run_specs.conf` containing example run spec entries? This will help users discover what tasks and subtasks are available. This will alse cause GitHub actions to test that your run spec function correctly parses and run spec entries. You can look at [`biomedical_run_specs.conf`](https://github.com/stanford-crfm/helm/blob/v0.2.3/src/helm/benchmark/presentation/run_specs_biomedical.conf) for an example.

Reply:
No problem. In the latest commits, we add `run_specs_cleva_v1.conf` (we add a `v1` tag for future compatibility), which lists all tasks / subtasks and `prompt_id` available.